src/extension.ts

```diff
@@ -18,7 +19,13 @@ export function activate(context: vscode.ExtensionContext) {
 
   // Check if there is a secret key
   const config = vscode.workspace.getConfiguration('faunadb');
-  const adminSecretKey = config.get<string>('adminSecretKey');
+  let adminSecretKey = config.get<string>('adminSecretKey');
```

Comment:
@BrunoQuaresma If we are going to move away from having the secret in the VSCode settings, do we still want to do this? I would imagine not unless it's a concern about backwards compatibility.